build.sbt

```diff
@@ -268,6 +313,42 @@ lazy val sourcing = project
     ),
     Test / fork          := true
   )
+
+lazy val sourcingNew = project
+  .in(file("sourcing-new"))
```

Comment:
the code should be placed under delta/sourcing with the appropriate package name